src/pages/text/team-chat-api/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+weight: 80
+category: "text"
+title: Overview
+tagline: "Build messaging software with Team Chat API by Text."
+desc: "API for a two-way communication between users. Available as a Web API."
+---
+
+# Introduction
+
+Team Chat API by Text is a backend service that allows for a two-way communication between users using the HTTP protocol (Web reference). It supports the communication between agents/team members, not between customers and agents.
+
+Team Chat API won't let you customize the LiveChat application, but it will let you build your own chat application from scratch.
+
+## Features
+
+- Communication can happen on public channels or via direct messages
+- Storing messaging history
+- Users can see messages that were sent before they joined the channel
+
+## Web reference
+
+The Web Reference section contains information about the Team Chat API's RESTful API and is available in the OpenAPI format.
+
+<SectionLink to={"/text/team-chat-api/web-reference"}>
+  See Web Reference
+</SectionLink>
+
+## FAQ
+
+### Is Team Chat API the same as Agent Chat API?
+
+No. Team Chat API is a separate service independent of the LiveChat product.
+
+### Can I use Team Chat API to chat with customers?
+
+Team Chat API was designed for building custom chat applications for internal communication (agent - agent; team member - team member).
```

Comment:
> (agent - agent; team member - team member)
I don't understand why we use both names together. Perhaps we should also mention bots here because it's one of the main use-cases, wdyt?

Reply:
I used both names to give more examples because I feel like "agent" is very business-oriented while "team member" better reflects the internal communication use case. Sure, I can add bots

Reply:
Yes but I mean that using "team member" after ";"  looks like it is something different. How about putting "team member" in brackets?